mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -206,6 +206,10 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 3 x
 		ARG 4 y
 		ARG 5 color
+	METHOD method_29343 drawOutline (IILjava/util/function/BiConsumer;)V
```

Comment:
```suggestion
	METHOD method_29343 drawWithOutline (IILjava/util/function/BiConsumer;)V
```
Since this method supplies the original position as well as outline positions, `drawWithOutline` is better imo.